app/src/main/java/com/example/taskmaster/MainActivity.java

```diff
@@ -187,7 +182,7 @@ public void handleMessage(Message inputMessage) {
                 }
 
                 mainActivityInstance.tasks = database.taskDao().getAll();
-                RecyclerView recyclerView = findViewById(R.id.mainRecyclerView);
+                recyclerView = findViewById(R.id.mainRecyclerView);

```

Comment:
mainActivityInstance.recylerView is how we access the recycler view from this other class
findViewById is a method on mainActivityInstance
mainActivtyInstance.findViewById